src/views/order_view.js

```diff
@@ -0,0 +1,43 @@
+import Backbone from 'backbone';
+import Order from '../models/order';
+
+const OrderView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+
+    this.listenTo(this.model.get('quote'), "change", this.processOrder);
+  },
+  processOrder(){
+    if(this.model.get('buy')){
+      if(this.model.buyIt()){
+        this.model.destroy();
+        this.remove();
+      }
+    }
+    else{
+      if(this.model.sellIt()){
+        if(this.model.sellIt()){
+          this.model.destroy();
```

Comment:
You have the same condition here twice! I suspect this is why sell orders don't clear themselves.